cbor/src/main/java/com/fasterxml/jackson/dataformat/cbor/CBORParser.java

```diff
@@ -5,15 +5,19 @@
 import java.math.BigInteger;
 import java.nio.charset.Charset;
 import java.nio.charset.StandardCharsets;
-import java.util.*;
+import java.util.ArrayList;
```

Comment:
This is ok, but going forward ideally let's leave imports as they are (unless need to add/remove of course), to avoid bogus diffs.

Reply:
My bad, I noticed while reviewing but for whatever reason I forgot to undo the change. I will avoid these going forward.